vstsBuild/vstsBuild.psm1

```diff
@@ -18,7 +17,7 @@ function Invoke-VstsPublishBuildArtifact
 {
     $ErrorActionPreference = 'Continue'
     $filter = Join-Path -Path (Get-Destination) -ChildPath '*'
-    log "Publishing artifacts: $filter"
+    Write-VstsMessage -message "Publishing artifacts: $filter"
```

Comment:
`Write-VstsMessage` uses `error` as the default value for `-type`, so this message will be written out as an error. That doesn't seem to be right.